server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SonarQubeMonitor.java

```diff
@@ -19,30 +19,45 @@
  */
 package org.sonar.server.platform.monitoring;
 
+import com.google.common.base.Function;
```

Comment:
who's using the information supposed to be deleted in [the related ticket](https://jira.sonarsource.com/browse/SONAR-7482)